connect/src/com/telenor/connect/id/ConnectIdService.java

```diff
@@ -58,6 +58,14 @@ public String getAccessToken() {
         return retrieveTokens().getAccessToken();
     }
 
+    public Date getAccessTokenExpirationTime() {
+        if (retrieveTokens() == null) {
```

Comment:
Yes, but you should probably do something like
```
DataType datatype = retrieveTokens();
if(datatype == null){
//...

return datatype.getExpirationDate();
```

ps. I don't know what datatype should be used.